src/js/main.js

```diff
@@ -1,87 +1,96 @@
+        "url": url,
+        "location": $('input[name="location"]:checked').val(),
+        "tags": tags,
+        "likes": 1
+      };
+      userCats.push(newCat);
+      $('.results').empty().append(drawPage());
+      $('#addCatModal').closeModal();
   }
+}
 
-  function uploadCat(url) {
-    firebase.database().ref('photos/' + $('#title').val()).set({
-      title: $('#title').val(),
-      url: url,
-      location: $('input[name="location"]:checked').val(),
-      tags: tags,
-      likes: 1
-    });
-  };
+$('form').on("submit", function(e){
+  addACat(e);
 });
 
-function writeLikes(title) {
-  firebase.database().ref('photos/' + title + '/likes').transaction(function(currentLikes) {
-    return currentLikes + 1;
-  });
-}
+//function writeLikes(title) {
+//  firebase.database().ref('photos/' + title + '/likes').transaction(function(currentLikes) {
+//    return currentLikes + 1;
+//  });
+//}
 
-function showData(photo, identifier) {
+function showData(photo) {
+  var identifier = photo.title.replace(/\s+/g, '');
   var html = '';
-      html += '<div class="col s3 m4 s12"><div class="card ' + identifier + '">';
-      html += '<div class="card-image">';
-      html += '<img class="materialboxed" src="' + photo.url + '" />';
-      html += '<span class="card-title">' + photo.title + '</span>';
-      html += '</div><div class="card-content">';
-      html += '<div class="location"><i class="fa fa-map-marker"></i><p>' + photo.location + '</p></div>'
-
-      for (let t = 0; t < photo.tags.length; t++) {
-        html += '<span class="chip blue darken-2">' + photo.tags[t] + '</span>';
-      }
-
-      html += '</div>';
-      html += `<div class="card-action">`
-      html += `<a class="waves-effect waves-light waves-blue blue darken-3 btn like"><i class="fa fa-thumbs-up"></i> <span class="likes_number">${photo.likes}</span></a>`;
-      html += `</div>`;
-      html += '</div></div>';
+    html += '<div class="col s3 m4 s12"><div class="card ' + identifier + '">';
+    html += '<div class="card-image">';
+    html += '<img class="materialboxed" src="' + photo.url + '" />';
+    html += '<span class="card-title">' + photo.title + '</span>';
+    html += '</div><div class="card-content">';
+    html += '<p>Location: ' + photo.location + '</p>';
+for(let t = 0; t <photo.tags.length; t++){
+    html += '<span class="chip blue darken-2">' + photo.tags[t] + '</span>';
+}
+    html += '</div>';
+    html += `<div class="card-action">`;
+    html += `<button class="waves-effect waves-light waves-blue blue darken-3 btn like"><i class="fa fa-thumbs-up"></i> <span class="likes_number">${photo.likes}</span></button>`;
+    html += `</div>`;
+    html += '</div></div>';
   return html;
 }
-
 $(document).ready(function(){
+  // render cards on load
+//  drawPage();
   // Materialize initalization
   $(".button-collapse").sideNav();
   $('.modal-trigger').leanModal();
   $('select').material_select();
+  $(".like").click(function(){
```

Comment:
I had this working in codepen, but I couldn't get it to work locally. I think my cache is holding on to some old stuff